plugins/parsers/registry.go

```diff
@@ -57,9 +57,20 @@ type Config struct {
 
 	// TagKeys only apply to JSON data
 	TagKeys []string
+	// FieldKeys only apply to JSON
+	StringFields []string
 	// MetricName applies to JSON & value. This will be the name of the measurement.
 	MetricName string
 
+	//holds a gjson path for json parser
```

Comment:
When adding comments, add a space before starting comment text (`// This is my comment`)